app/views/dashboard/_additional_kyc_info_banner.njk

```diff
@@ -1,29 +1,48 @@
 {% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
 
-{% if (isAdminUser and requiresAdditionalKycData and activeCredential.payment_provider === 'stripe') %}
+{% if (requiresAdditionalKycData and activeCredential.payment_provider === 'stripe') %}
 
   <div class="govuk-grid-column-full">
+
     {% set html %}
-      <p class="govuk-notification-banner__heading">
-          Add more information by {{ stripeKycAdditionalDataDueDate }}
-      </p>
-      <p class="govuk-body">
-          Updated Know Your Customer (KYC) anti-money laundering regulations mean that you need to add more details about your organisation.
-      </p>
-      <p>
-          You have until {{ stripeKycAdditionalDataDueDate }} to add these details. If you do not add them, your service will stop taking payments and Stripe will stop paying out to your bank account.
-      </p>
-      <p class="govuk-body">
-        Continue to
-        <a id="add-additional-kyc-details" class="govuk-notification-banner__link"
-           href="{{ formatAccountPathsFor(routes.account.yourPsp.index, currentGatewayAccount.external_id, activeCredential.external_id) }}">
-          Add new KYC Information</a>.
-      </p>
+        
+        <h2 class="govuk-notification-banner__heading">
```

Comment:
(minor) Any reason this is a h2 ? It is making both notification title and this text h2

![image](https://user-images.githubusercontent.com/40598480/150833678-aab52d81-f76d-4bb5-8db8-8dc1b378b4cf.png)

Reply:
My reasoning was that the first `p` was classed `banner__heading` and should probably be a heading element (the design system uses a mix of heading or paragraph tags for this class).

I can revert back / use a lower order heading if preferable?

Reply:
component has two elements (header & heading). `banner__header` is h2 and `banner__heading` is usually a paragraph with styling to emphasize text.

Reply:
yeah can use paragraph so it will be inline with design system.

Reply:
updated